keybert/backend/__init__.py

```diff
@@ -1,3 +1,4 @@
 from ._base import BaseEmbedder
+from ._sentencetransformers import SentenceTransformerBackend
 
 __all__ = ["BaseEmbedder"]
```

Comment:
```suggestion
__all__ = ["BaseEmbedder", "SentenceTransformerBackend"]
```

By adding it to `__all__` it can be accessed directly as follows:

```python
from keybert.backend import SentenceTransformerBackend
```